config/schedule.yml

```diff
@@ -12,4 +12,9 @@ trigger_scheduled_items_job:
   cron: "*/5 * * * *"
   class: "TriggerScheduledItemsJob"
   queue: scheduled_jobs
-  
\ No newline at end of file
+
+# executed At 00:00 every night
+account_conversations_resolution_scheduler_job:
```

Comment:
rather than having a separate schedule for this job. let's run it inside the "TriggerScheduledItemsJob" 

- We will have one job that triggers all the routine maintenance jobs
- Also conversations will be resolved through out the day by the system, rather than as one batch at start of every day.